backend/src/main/java/ddangkong/domain/balance/room/Room.java

```diff
@@ -3,27 +3,51 @@
 import ddangkong.exception.BadRequestException;
 import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import lombok.AccessLevel;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 
 @Entity
 @Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class Room {
 
     private static final int DEFAULT_TOTAL_ROUND = 5;
+    private static final int DEFAULT_TIME_LIMIT = 10000;
```

Comment:
사소한 의견) 조금 더 보기 편하지 않나요? (반영 안하셔도 됩니다~)
```suggestion
    private static final int DEFAULT_TIME_LIMIT = 10_000;
```